test/support/helpers.js

```diff
@@ -0,0 +1,10 @@
+export function suppressConsoleErrors () {
+  beforeEach(() => {
+    jest.spyOn(console, 'error');
+    console.error.mockImplementation(() => {});
+  });
+
+  afterEach(() => {
+    console.error.mockRestore();
+  });
+}
```

Comment:
This method deactivates the virtual console error method. This is useful when we expect a test to produce errors.

Reply:
It might be useful to add this as a comment in the code as well

Reply:
Good call -- done!